samples/Exporters/TestPrometheus.cs

```diff
@@ -31,22 +32,41 @@ internal class TestPrometheus
         internal static object Run()
         {
             var promOptions = new PrometheusExporterOptions() { Url = "http://localhost:9184/metrics/" };
-            Metric<long> metric = new Metric<long>("sample");
-            var promExporter = new PrometheusExporter<long>(promOptions, metric);
+            var promExporter = new PrometheusExporter(promOptions);
+            var simpleProcessor = new UngroupedBatcher(promExporter, TimeSpan.FromSeconds(5));
+            var meter = MeterFactory.Create(simpleProcessor).GetMeter("library1") as MeterSDK;
+            var testCounter = meter.CreateInt64Counter("testCounter");
+
+            var labels1 = new List<KeyValuePair<string, string>>();
+            labels1.Add(new KeyValuePair<string, string>("dim1", "value1"));
+
+            var labels2 = new List<KeyValuePair<string, string>>();
+            labels2.Add(new KeyValuePair<string, string>("dim1", "value2"));
+
             try
-            {                                
+            {
                 promExporter.Start();
-                List<KeyValuePair<string, string>> label1 = new List<KeyValuePair<string, string>>();
-                label1.Add(new KeyValuePair<string, string>("dim1", "value1"));
-                var labelSet1 = new LabelSet(label1);
-                metric.GetOrCreateMetricTimeSeries(labelSet1).Add(100);
-                Task.Delay(30000).Wait();
-                metric.GetOrCreateMetricTimeSeries(labelSet1).Add(200);
-                Console.WriteLine("Look at metrics in Prometheus console!");
-                Console.ReadLine();
+
+                for (int i = 0; i < 1000; i++)
+                {
+                    testCounter.Add(SpanContext.BlankLocal, 100, meter.GetLabelSet(labels1));
+                    testCounter.Add(SpanContext.BlankLocal, 10, meter.GetLabelSet(labels1));
+                    testCounter.Add(SpanContext.BlankLocal, 200, meter.GetLabelSet(labels2));
+                    testCounter.Add(SpanContext.BlankLocal, 10, meter.GetLabelSet(labels2));
+
+                    if (i % 10 == 0)
+                    {
+                        // Collect is called here explicitly as there is 
+                        // Controller implementation yet.
+                        meter.Collect();
```

Comment:
Since it's temporary - the code will look better if it will cast to `MeterSDK` here, not on creation. And add `TODO:` to replace it with the proper way doing it in future